ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/SearchTreeRaoResultExporters.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.search_tree_rao;
+
+import com.farao_community.farao.commons.FaraoException;
+import com.google.common.base.Suppliers;
+import com.powsybl.commons.util.ServiceLoaderCache;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.*;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+/**
+ * @author Pengbo Wang {@literal <pengbo.wang at rte-international.com>}
+ */
+public final class SearchTreeRaoResultExporters {
+
+    private static final Supplier<List<SearchTreeRaoResultExporter>> SEARCH_TREE_RAO_RESULTS_EXPORTERS
+            = Suppliers.memoize(() -> new ServiceLoaderCache<>(SearchTreeRaoResultExporter.class).getServices())::get;
+
+    private SearchTreeRaoResultExporters() {
+    }
+
+    public static void exportSearchTreeRaoResult(SearchTreeRaoResult result, String format, OutputStream outputStream) throws IOException {
```

Comment:
I think you can remove throws IOException

Reply:
ok